### ISSUE TITLE
[Feature request] Sync logo for Tag

### ISSUE THREAD

**Issue by FTOcotral** [ForkIssues/TrackerWin#1583]
It would be a nice feature if we can have a sync logo beside a tag, just like branches. Because it happen a lot that someone miss the push tag to server as it is not obviously seen on the main panel.
Thanks

**Comment by DanPristupov** [comment]
> It would be a nice feature if we can have a sync logo beside a tag, just like branches.

Yes, but tags in git are purely local with the ability to push them.

In other words, there's no such thing as 'remote tag'.

**Comment by FTOcotral** [comment]
> > It would be a nice feature if we can have a sync logo beside a tag, just like branches.
> 
> Yes, but tags in git are purely local with the ability to push them.
> 
> In other words, there's no such thing as 'remote tag'.

What about "git ls-remote --tags" ?

**Comment by DanPristupov** [comment]
Tags are not as simple as they look. For example, in practice it's almost impossible to remove a tag on a remote server.


Regarding `git ls-remote --tags`:

Let's say a repository has 3 remotes: `origin1`, `origin2` and `origin3`.
There is a local tag `tag1`.

`tag1` is NOT pushed to `origin1`
`tag1` hasn't been pushed to `origin2`
`tag1` hasn't been pushed to `origin3`, but was created here independently by some other user on a different commit

What logo do you expect to see?

**Comment by FTOcotral** [comment]
You can have the same case with branches. One branch can be push to a remote but not on another one. 
The logo could be the same logo as the one we have on remote branches. 
![image](https://user-images.githubusercontent.com/93710945/177801233-b12e056d-a8c4-48b9-8f37-1dc4292c7bdf.png)

**Comment by DanPristupov** [comment]
This is a label of a remote branch. When a remote branch is named same as the local branch (master), its name is trimmed (which is optional, btw). But it's still a separate reference (a separate branch object). As I said, there's no such thing as a remote tag reference.

**Comment by FTOcotral** [comment]
So, is there any way in fork to get the same information as a "git ls-remote --tags" ?

**Comment by DanPristupov** [comment]
No, currently there's no UI for that.
